src/formElementPlugins/listBuilder/src/types.ts

```diff
@@ -8,6 +8,26 @@ export enum DisplayType {
   LIST = 'list',
 }
 
+export interface ListBuilderParameters {
+  label?: string
+  description?: string
+  createModalButtonText?: string
+  modalText?: string
+  addButtonText?: string
+  updateButtonText?: string
+  deleteItemText?: string
+  inputFields: TemplateElement[]
+  displayFormat?: { title: string; subtitle: string; description: string }
+  displayType?: DisplayType
+  default?: ResponseFull
+  inlineOpen?: boolean
+  tableExcludeColumns?: string[]
+  // These affect mobile viewing only
+  hideFromMobileTableIfEmpty?: true | string[]
+  minMobileTableLabelWidth?: number | string
+  maxMobileTableLabelWidth?: number | string
+}
```

Comment:
We were pretty sloppy with our use of Typescript in the early days (lots of `any` around the place), but I'm slowly but surely properly defining types where I can going forward.